x-pack/plugins/spaces/server/lib/copy_to_spaces/copy_to_spaces.ts

```diff
@@ -39,7 +39,7 @@ export function copySavedObjectsToSpacesFactory(
       excludeExportDetails: true,
       objects: options.objects,
       savedObjectsClient,
-      types: eligibleTypes,
+      types: options.objects.length > 0 ? undefined : eligibleTypes,
```

Comment:
As explained in a previous comment, When sending both `objects` and `types`, only `objects` were used. However the validation to not send both was performed in the route handler. As I moved it to the service, I had to adapt some calls to no longer send both.

Reply:
@legrego do you happen to recall why we used to always be passing in `eligibleTypes` here? It looks like this was being ignored... I'm tempted to say we should just always pass in `undefined` here if that's the case.

Reply:
The intent was to prevent namespace agnostic types from being specified, but that's handled on the subsequent call to `importSavedObjects`, so I agree we should just always pass `undefined` here, rather than `eligibleTypes`

Reply:
Ack, will just remove the `types` parameter from the call then